code/images/SCIENCE_files/bootstrap.min.css

```diff
@@ -0,0 +1,7 @@
+@charset "UTF-8";/*!
```

Comment:
I'm having a hard time reading this code since it's all in one row, but it looks good from what I can see! If you use bootstrap again, maybe just put it in a column like the index-file? :) If thats possible